zlibWrapper/Makefile

```diff
@@ -7,8 +7,9 @@
 
 # Paths to static and dynamic zlib and zstd libraries
 ifneq (,$(filter Windows%,$(OS)))
-STATICLIB = ../../zlib/libz.a ../lib/libzstd.a
-IMPLIB    = ../../zlib/libz.dll.a ../lib/libzstd.a
+ZLIBDIR = ../../zlib1.2.8
```

Comment:
Fair point.
The initial intention was to support it, but the goal was postponed because there used to be no way to test it properly.

Maybe that action could be resurrected thanks to the great improvements you brought to CI test suite. If CygWin target can be selected, then it becomes possible to test compilation and run.

In any case, don't consider it a primary objective. It's more a "nice to have" target for the time being.